src/Sarif.UnitTests/Core/PropertyBagHolderTests.cs

```diff
@@ -9,7 +9,7 @@
 using Newtonsoft.Json;
 using Newtonsoft.Json.Serialization;
 
-namespace Microsoft.CodeAnalysis.Sarif.Core
+namespace Microsoft.CodeAnalysis.Sarif
```

Comment:
>Sarif [](start = 33, length = 5)

PropertyBagHolder is in Core.

IMO the convention should be that tests for classes in namespace X.Y.Z should be in namespace X.Y.Z.Tests, but I don't know if we've done that.

TL;DR, why bother removing this?